src/player.h

```diff
@@ -979,8 +982,14 @@ class player : public Character, public JsonSerializer, public JsonDeserializer
         bool pda_cached;
 
         // Drench cache
-        std::map<int, std::map<std::string, int> > mMutDrench;
-        std::map<body_part, int> mDrenchEffect;
+        enum water_tolerance {
+            WT_IGNORED = 0,
+            WT_NEUTRAL,
+            WT_GOOD,
+            NUM_WATER_TOLERANCE
+        };
+        std::array<std::array<int, NUM_WATER_TOLERANCE>, num_bp> mut_drench;
+        std::array<int, num_bp> drench_capacity;
```

Comment:
`drench_capacity` appears to be constant. Maybe it should be made static? Or are there plans to change it depending player state (e.g. mutations/bionics)?